packages/manager/modules/pci/src/projects/new/payment/controller.js

```diff
@@ -361,6 +364,152 @@ export default class PciProjectNewPaymentCtrl {
+              (ua.match(/tablet/i) && !ua.match(/RX-34/i)) ||
+              ua.match(/FOLIO/i)
+            ) {
+              return 'tablet';
+            }
+            if (
+              ua.match(/Linux/i) &&
+              ua.match(/Android/i) &&
+              !ua.match(
+                /Fennec|mobi|HTC.Magic|HTCX06HT|Nexus.One|SC-02B|fone.945/i,
+              )
+            ) {
+              return 'tablet';
+            }
+            if (
+              ua.match(/Kindle/i) ||
+              (ua.match(/Mac.OS/i) && ua.match(/Silk/i))
+            ) {
+              return 'tablet';
+            }
+            if (
+              ua.match(
+                /GT-P10|SC-01C|SHW-M180S|SGH-T849|SCH-I800|SHW-M180L|SPH-P100|SGH-I987|zt180|HTC(.Flyer|_Flyer)|Sprint.ATP51|ViewPad7|pandigital(sprnova|nova)|Ideos.S7|Dell.Streak.7|Advent.Vega|A101IT|A70BHT|MID7015|Next2|nook/i,
+              ) ||
+              (ua.match(/MB511/i) && ua.match(/RUTEM/i))
+            ) {
+              return 'tablet';
+            }
+            if (
+              ua.match(
+                /BOLT|Fennec|Iris|Maemo|Minimo|Mobi|mowser|NetFront|Novarra|Prism|RX-34|Skyfire|Tear|XV6875|XV6975|Google.Wireless.Transcoder/i,
+              )
+            ) {
+              return 'mobile';
+            }
+            if (
+              ua.match(/Opera/i) &&
+              ua.match(/Windows.NT.5/i) &&
+              ua.match(/HTC|Xda|Mini|Vario|SAMSUNG-GT-i8000|SAMSUNG-SGH-i9/i)
+            ) {
+              return 'mobile';
+            }
+            if (
+              (ua.match(/Windows.(NT|XP|ME|9)/) && !ua.match(/Phone/i)) ||
+              ua.match(/Win(9|.9|NT)/i)
+            ) {
+              return 'desktop';
+            }
+            if (ua.match(/Macintosh|PowerPC/i) && !ua.match(/Silk/i)) {
+              return 'desktop';
+            }
+            if (ua.match(/Linux/i) && ua.match(/X11/i)) {
+              return 'desktop';
+            }
+            if (ua.match(/Solaris|SunOS|BSD/i)) {
+              return 'desktop';
+            }
+            if (
+              ua.match(
+                /Bot|Crawler|Spider|Yahoo|ia_archiver|Covario-IDS|findlinks|DataparkSearch|larbin|Mediapartners-Google|NG-Search|Snappy|Teoma|Jeeves|TinEye/i,
+              ) &&
+              !ua.match(/Mobile/i)
+            ) {
+              return 'desktop';
+            }
+            return 'mobile';
+          },
+        };
+      }
+
+      if (typeof window.tC.msr !== 'object') {
+        window.tC.msr = [];
+      }
+      window.tC.msr.dns = 'ovh.commander1.com';
+      window.tC.msr.id_site = '3810';
+      window.tC.msr.idorder =
+        Date.now().toString(36) +
+        Math.random()
+          .toString(36)
+          .substring(2, 12)
+          .padStart(12, 0);
+      window.tC.msr.amount = '{{order_amount_tax_free}}';
+      window.tC.msr.currency = '{{currency code}}';
+      window.tC.msr.user_id = '{{customer ID}}';
```

Comment:
**nitpick:** The values in the single quotes must be dynamic values based on the logged in user and the total mount.